dts/arm/nxp/nxp_rt5xx_common.dtsi

```diff
@@ -455,6 +455,7 @@
 
 &flexspi {
 	compatible = "nxp,imx-flexspi";
+	status = "disabled";
```

Comment:
why is this disabled, at least add comment in commit message.  Something like 'flexspi will be board specific modules of FLASH so leave it to the board to enable...'

Reply:
Updated commit message, as you pointed out this is disabled at the SOC level because the external flash module is board specific and needs to be tested on a per board basis.